river/cluster/clustream.py

```diff
@@ -32,6 +32,11 @@ class CluStream(base.Clusterer):
     the use of Welford's algorithm [^2] to calculate the incremental variance, facilitated
     through `stats.Var()` available within `River`.
 
+    Since `River` does not support an actual "off-line" phase of the clustering algorithm (as data points
```

Comment:
```suggestion
    Since River does not support an actual "off-line" phase of the clustering algorithm (as data points
```

Reply:
Marked as resolved since `time_gap` is now omitted.

Reply:
@hoanganhngo610 not for this change right?

Reply:
Not for this change. This has been fixed in the latest commit below!